documentation/docs/30-advanced/10-advanced-routing.md

```diff
@@ -227,9 +225,41 @@ src/routes/
 └ +layout.svelte
 ```
 
+#### Breaking out of the root layout
+
+There is no way to break out of the root layout. You can be sure that it's always present in your app and for example put app-wide UI or behavior in it - and if you don't define one, it falls back to a simple `<slot />` component. If you want some pages to _not_ have the same layout as the rest, then you have two options:
+
+Option 1: Create a `(group)` at the root and a `+layout.svelte` within it, which is used for each page below it. Every page that should inherit a blank root layout instead is put outside of that root `(group)`. In the following example, the login page inherits a blank root layout, while the other pages inherit the layout inside `(group)`:
+
+```
+src/routes/
+├ (group)
+│ ├ about/
+│ │ └ +page.svelte // inherits (group) layout
+│ └ +layout.svelte
+├ login/
+│ └ +page.svelte // inherits blank root layout
+```
+
+Option 2: Add an exception to your root `+layout.svelte` where you don't render the surrounding UI on specific pages:
+
+```svelte
+/// file: src/routes/+layout.svelte
+<script>
+	import { page } from '$app/stores';
+</script>
+
+{#if $page.url.pathname !== '/login'}
+	<slot />
+{:else}
+	<div>Surrounding UI</div>
+	<slot />
+{/if}
+```
```

Comment:
I feel like this is a pattern we don't really want to promote — it's the sort of thing that can get out of hand quite quickly, and it introduces sort of a mismatch between the UI code (which has `if` statements) and data fetching code (which doesn't). It also defeats code-splitting — in a more realistic example, one of the branches would use a component, which we'd be importing in both cases

Reply:
For simplicity's sake I think this solution is actually perfectly fine for a number of use cases. I agree though that this gets out of hand at some point, so we should point that out.

Reply:
Just remembered this bug: https://github.com/sveltejs/kit/issues/2527

We absolutely don't want to encourage this pattern